src/collections/_documentation/data-management/rollups.md

```diff
@@ -74,3 +70,200 @@ A generic error, such as a database connection error, has many different stack t
 The following example will just completely overwrite Sentry's grouping by omitting `{% raw %}{{ default }}{% endraw %}` from the array:
 
 {% include components/platform_content.html content_dir='fingerprint-database-connection-example' %}
+
+## Grouping Algorithm Versioning
+
+When a Sentry project is created the latest and greatest version of the grouping algorithm is automatically selected.  However we will improve the grouping system going forward but only new projects will see these improvements not to create new groups for you.  If you do want to upgrade an existing project to a new grouping algorithm version you can do so in the project settings.  Note that you can only upgrade.  To downgrade you will need to contract customer support.
+
+## Grouping Enhancements and Server Side Fingerprinting
+
+When the default grouping does not yield satisfying results there are ways in which grouping can be influenced from the server through grouping enhancements and server side fingerprinting.
```

Comment:
You need to scroll down to learn more about the fingerprinting enhancements. What about to mention in high level in this introduction what about the enhancements are. Something like this: "With Grouping enhancements you can decide what data from stack trace will be used for grouping. With the fingerprinting enhancements you can define your own groups based on the stack trace data , exception value or message."